rasa/nlu/training_data/loading.py

```diff
@@ -173,6 +178,17 @@ def guess_format(filename: Text) -> Text:
                 guess = fformat
                 break
 
+    if guess == UNK:
+        try:
+            from rasa.nlu.training_data.formats.rasa_yaml import KEY_NLU
+
+            content = io_utils.read_yaml_file(filename)
+            if KEY_NLU in content:
+                guess = RASA_YAML
+        # YAML lib isn't exposing all necessary Errors/Warnings
+        except Exception:
+            pass
```

Comment:
Would it be possible to maybe find if there is a parent exception for all YAML-related errors and use that one? Possibly by looking into the code of the library itself.

Reply:
I did try this, but YAML lib isn't exposing all necessary Errors/Warnings. 
They wrap some errors as "YamlWarnings" and never expose them, no idea why.

Reply:
Fixed that - I was wrong.